eventdata/challenges/elasticlogs-querying.json

```diff
@@ -1,3 +1,5 @@
+{% set p_time_period = (time_period | default(1800)) %}
```

Comment:
IMHO the parameter name should be more specific because it's hard to understand where this parameter gets applied just by looking at the name. Also, we might want to introduce similar parameters for other challenges as well and then it might be unclear what "time period" we mean. How about `query_time_period`? 

I think for this very simple challenge it would also be ok if we just use the variable inline without assigning it to another internal one.

Reply:
ill rename and leave it at the top so i can use it in the description as well